docs/getting_started.rst

```diff
@@ -7,21 +7,34 @@ Getting Started
 Installing RTI Connector for Python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are two ways to obtain *RTI Connector* for Python. You can install it with
-*pip*:
+There are two ways to obtain *RTI Connector* for Python: with
+*pip* or by cloning the repository.
+
+**To install Connector with pip:**
 
 .. code:: bash
 
    $ pip install rticonnextdds_connector
 
+The above command installs the latest version of *Connext* by default. To
```

Comment:
Obviously, these are not the right instructions. I do not know what the right instructions are. Elaine told me Connector-py needs the same changes that we made to Connector-js and to the Connext Python API regarding versions.

Reply:
The pip documentation is accurate, but I'd prefer using `rticonnextdds-connector` (both are valid) so that it matches JS documentation.